packages/coordinator/src/services/validation/claim-validator.ts

```diff
@@ -60,5 +61,12 @@ export function verifyClaim(
     return permissionErrors
   }
 
+  const medicationDispenseEndorsement = getMedicationDispenseEndorsement(claim)
+  if (!medicationDispenseEndorsement.length) {
```

Comment:
This logic could be included in the getMedicationDispenseEndorsement function. Then you could call it medicationDispenseEndorsementPresent and return a boolean.

Reply:
The medicationDispenseEndorsementPresent function has been added.